src/System.Private.CoreLib/Resources/Strings.resx

```diff
@@ -3664,4 +3664,10 @@
   <data name="IndexOutOfRange_ArrayWithOffset" xml:space="preserve">
     <value>ArrayWithOffset: offset exceeds array size.</value>
   </data>
-</root>
+  <data name="Serialization_DangerousDeserialization" xml:space="preserve">
+    <value>An action was attempted during deserialization that could lead to a security vulnerability. The action has been aborted.</value>
+  </data>
+  <data name="Serialization_DangerousDeserialization_Switch" xml:space="preserve">
+    <value>An action was attempted during deserialization that could lead to a security vulnerability. The action has been aborted. To allow the action under Serialization Guard, set the '{0}' AppContext switch to true.</value>
```

Comment:
Can we remove the "under Serialization Guard" piece?  Are we actually using that name as an official, branded name for a piece of technology?

Reply:
It's not clear what branding we'll do, removed "under Serialization Guard"